project/BuildSettings.scala

```diff
@@ -18,7 +18,7 @@ object BasicSettings extends AutoPlugin {
 
   override def projectSettings: Seq[Def.Setting[_]] = Seq(
     organization := "org.hatdex",
-    version := "2.6.7",
+    version := "2.6.10-SNAPSHOT",
```

Comment:
`SNAPSHOT` deprecated on @benshaw suggestion. I also think we will find it futile to keep HAT and DEX client library versions aligned all the time. What you think @benshaw?

Reply:
Agree probably no point keeping HAT and DEX versions aligned

SNAPSHOT is ok when doing local dev but i dont think we should be pushing to the repo with it as it does not correlate to a single version but instead the latest iteration with that version number which could make debugging troublesome